src/odemis/gui/xmlh/resources/panel_tab_fastem_acqui.xrc

```diff
@@ -72,6 +72,13 @@
                           <flag>wxALL|wxEXPAND</flag>
                           <border>10</border>
                         </object>
+                        <object class="sizeritem">
+                          <object class="wxCheckBox" name="chk_beam_blank_off">
+                            <label>Turn off beam blanking in between fields</label>
```

Comment:
It feels like double negation... which is negated if you don't check the box. It's a little confusing.
How about "Do not blank beam in between fields" or even "Blank beam in between fields" (in which case the check box has to be inverted)?